### DISCUSSION TITLE
Forced nucleation with R&S friction law?

### DISCUSSION THREAD

**Discussion by francescomosconi** [SeisSol/SeisSol#933] [Q&A] [answered]
Hi everyone!
Is there a way to nucleate the rupture without over-stressing the fault adopting the R&S friction law?
I'm referrring to something like the FL=16 with the forced nucleation function.
Many thanks in advance!

**Comment by Thomas-Ulrich** [comment, marked as answer]
Hi,
No.
(but you could also try to decrease effective stress instead of increase shear stress).

**Comment by francescomosconi** [comment]
Many thanks for your reply.
Francesco
